server/jobs/slips.py

```diff
@@ -0,0 +1,106 @@
+import math
+import io
+import csv
+from collections import defaultdict
+from datetime import datetime as dt
+
+from server import jobs
+from server.models import Assignment, ExternalFile
+from server.utils import encode_id, local_time, output_csv_iterable
+from server.constants import TIMESCALES
+
+timescales = {'days':86400, 'hours':3600, 'minutes':60}
```

Comment:
I did this originally, but realized if I would then have to use the dictionary keys to fill in the form options, which wouldn't have a guaranteed ordering.  

Now that I've thought about it, I think I'll just use an OrderedDict to preserve the ordering of the keys for display on the form.